contracts/AutDAO.sol

```diff
@@ -0,0 +1,53 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./daoUtils/abstracts/DAOMembers.sol";
+import "./daoUtils/abstracts/DAOInteractions.sol";
+import "./daoUtils/abstracts/DAOUrls.sol";
+import "./daoUtils/abstracts/AutIDAddress.sol";
+import "./daoUtils/abstracts/DAOMetadata.sol";
+import "./daoUtils/abstracts/DAOMarket.sol";
+import "./daoUtils/abstracts/DAOCommitment.sol";
+
+
+/// @title AutDAO
+/// @notice
+/// @dev
+contract AutDAO is DAOMembers, DAOInteractions, DAOMetadata, DAOUrls, DAOMarket, DAOCommitment {
+    
+    address private deployer;
+
+    /// @notice Sets the initial details of the DAO
+    /// @dev all parameters are required.
+    /// @param _deployer the address of the DAOTypes.sol contract
+    /// @param _autAddr the address of the DAOTypes.sol contract
+    /// @param _market one of the 3 markets
+    /// @param _metadata url with metadata of the DAO - name, description, logo
+    /// @param _commitment minimum commitment that the DAO requires
+    constructor(
+        address _deployer,
+        address _autAddr,
```

Comment:
Better to pass in as the type rather than as an address

Reply:
okie :)